### ISSUE TITLE
Custom JS not working anymore

### ISSUE THREAD

**Issue by wrzpic** [mikecrittenden/chrome-shortkeys#96]
We used shortcut "Ctrl+g" for an long time, but we think since Google Chrome update (?) the  shortcut "Ctrl+g" with following javascript doesn´t work.

javascript:__doPostBack('ctl00$ctl00$ctl00$ContentContainer$MainContent$MainContent$allPages','')

![javascript_shortcut](https://user-images.githubusercontent.com/34124249/33422869-6a90c1e0-d5b7-11e7-8308-8db7993c38c8.jpg)

Pls support our issue as soon as possible. THX

**Comment by mikecrittenden** [comment]
You should be able to remove the `javascript:` part. Try that first and let me know if that doesn't work.

**Comment by wrzpic** [comment]
We delete it and create a new javascript, it dosen´t work better. 👎 
When we type the code "javascript:__doPostBack('ctl00$ctl00$ctl00$ContentContainer$MainContent$MainContent$allPages','')" into field "**Javascript code to run**"  the shortcut option windows starts to flickering in horizontal axis
The flickering begins after writing 2nd code line...

**Comment by mikecrittenden** [comment]
Did you delete the javascript: part like I said?

**Comment by mikecrittenden** [comment]
The flickering is probably #86 which is a known issue at the moment when you have JS lines that wrap.

**Comment by wrzpic** [comment]
yes we delete JS and yes we have the same Issue you reported above =>  #86

**Comment by mikecrittenden** [comment]
@wrzpic I'm afraid you may be misunderstanding what I'm saying. I'm saying to paste this into the JS textbox:

```
__doPostBack(
'ctl00$ctl00$ctl00$ContentContainer$MainContent$MainContent$allPages',
'')
```

See if that works? Note that I completely removed the `javascript:` part of it (because that's from a very old version of the extension that breaks nowadays) and also added a couple line breaks so that the resizing thing wouldn't happen.

**Comment by wrzpic** [comment]
YES, it woks fine, Sorry for my understanding. THX

**Comment by mikecrittenden** [comment]
No worries! I'm in the middle of pushing a fix for #86 now, by the way.
